shared/js/initApp.js

```diff
@@ -1,157 +1,113 @@
 // TODO (brent) - way too many globals
 // TODO (brent) - I wonder if we should sub-namespace dashboard
-/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga*/
+/* global script_path, Dialog, CDOSounds, dashboard, appOptions, $, trackEvent, Applab, sendReport, cancelReport, lastServerResponse, showVideoDialog, ga, digestManifest*/
 
-var timing = require('./timing');
-var chrome34Fix = require('./chrome34Fix');
-dashboard.project = require('./project');
+window.setupApp = function () {
+  var timing = require('./timing');
+  var chrome34Fix = require('./chrome34Fix');
```

Comment:
I guess another option would be to have a single entry point outside of initApp (like a main.js) that requires initApp, and then initApp can have a module.exports.